app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java

```diff
@@ -215,31 +316,80 @@ public void onClick(DialogInterface arg0, int arg1) { //open up the gallery
                                 pictureActionIntent,
                                 GALLERY_PICTURE);
 
-
                     }
                 });
 
         myAlertDialog.setNegativeButton("Camera",
```

Comment:
I'm not sure if it's the emulator or something else but when I select the camera option and take the picture it does not go back to the app. I have to close the camera, retry adding a picture, and get it from the gallery.

Reply:
That is strange... it wasn't like this before, but I started trying this again. Sometimes it happens and sometimes it doesn't. We may want to revisit this again later to be safe. Good catch!